src/main/scala-2.11/markets/orders/orderings/PriceOrdering.scala

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2015 David R. Pugh, J. Doyne Farmer, and Dan F. Tang
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package markets.orders.orderings
+
+import markets.orders.Order
+
+
+trait PriceOrdering[T <: Order] extends Ordering[T] {
+
+  override def compare(order1: T, order2: T): Int = {
+    if (hasPricePriority(order1, order2)) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using case matching instead of else if blocks](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1425046383/issues/source?bid=2921362&fileBranchId=2987952#l24)